src/platform/i386/boot_comp.c

```diff
@@ -220,9 +232,11 @@ kern_boot_upcall(void)
 	u8_t *entry = mem_bootc_entry();
 	u32_t flags = 0;
 	void *p;
+	if (get_cpuid() >= 0) {
```

Comment:
blank line after declarations and before body.

Can `get_cpuid()` return -ve values? Isn't that an error condition? 
`chal_user_upcall` should not be called with it right? 
Just thinking `assert(get_cpuid() >= 0);` and no `if` condition would probably be the right way to go here.

Reply:
I will address this